src/ontology/mondo-edit.obo

```diff
@@ -172614,6 +172610,7 @@ is_a: MONDO:0018894 {source="DC-OMIM:158590", source="MONDO:Redundant"} ! distal
 is_a: MONDO:0024237 {source="MESH:C563561/inferred"} ! inherited neurodegenerative disorder
 intersection_of: MONDO:0018894 ! distal hereditary motor neuropathy
 intersection_of: has_material_basis_in_germline_mutation_in http://identifiers.org/hgnc/30171 ! HSPB8
+relationship: excluded_from_qc_check http://purl.obolibrary.org/obo/mondo/sparql/qc/mondo/qc-omim-subsumption.sparql
```

Comment:
@MeeSiing please add the reasons why you added this "excluded from QC check" annotation. 
I want to make sure that (1) you understand what the QC failure meant, (2) you have done the appropriate research and came to the correct conclusion that this QC check can be added, (3) this type of QC check exclusion must be documented.